File: build-system/integration-test/test-projects/singleAtom/atom/src/main/java/com/android/tests/singleatom/atom/MainActivity.java
package com.android.tests.singleatom.atom;

import android.app.Activity;
import android.os.Bundle;
import java.util.logging.Logger;


public class MainActivity extends Activity {
    /** Called when the activity is first created. */
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.atom_main);

        Logger.getLogger(Logger.GLOBAL_LOGGER_NAME).warning(Atom.someString());
    }
}

File: build-system/integration-test/test-projects/utp/app/src/main/kotlin/com/example/android/kotlin/MainActivity.kt
package com.example.android.kotlin

import android.app.Activity
import java.util.logging.Logger.getLogger

class MainActivity : Activity() {
    companion object {
        fun stubFuncForTestingCodeCoverage() {
            getLogger("MainActivity").info("stubFuncForTestingCodeCoverage()")
        }
    }
}
